webknossos/examples/create_dataset_from_images.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from webknossos import Dataset
+from webknossos.dataset import COLOR_CATEGORY
+
+INPUT_DIR = Path(__file__).parent.parent / "testdata" / "tiff"
+OUTPUT_DIR = Path("testoutput/my_tiff_dataset")
+
+
+def main() -> None:
+    """Convert a folder of image files to a WEBKNOSSOS dataset."""
+    dataset = Dataset.from_images(
+        input_path=INPUT_DIR,
+        output_path=OUTPUT_DIR,
+        voxel_size=(11, 11, 11),
+        layer_category=COLOR_CATEGORY,
+        compress=True,
+    )
+
+    print(f"Saved {dataset.name} at {dataset.path}.")
+
+    # dataset.upload()
```

Comment:
I would add another comment to this line. something like, "in case you want to upload your dataset straight away uncomment and run:"

Reply:
You probably also need to wrap the `dataset.upload` example in a `with webknossos_context(token=..., url=...)`